### PR TITLE
Add Czech Lang

### PR DESCRIPTION
**Pull request by halali**
#### Database Migration
NO

#### Description
Add Czech Language

### CODE REVIEW

**Comment by galli-leo** [comment]
Please add tests for the new language.

File: src/NzbDrone.Core/Parser/Language.cs [hunk 1]
```diff
@@ -28,7 +28,8 @@ public enum Language
         Greek = 20,
         Korean = 21,
         Hungarian = 22,
-        Hebrew = 23
+        Hebrew = 23,
+		Czech = 24
```
Comment (galli-leo) [inline src/NzbDrone.Core/Parser/Language.cs]: Please fix the whitespace here.

**Comment by halali** [comment]
Done

**Comment by Qstick** [comment]
@halali needs rebase

**Comment by halali** [comment]
@Qstick Done

**Pull request merged**